client/app/hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
out of curiosity what are these for? I removed them, refreshed the page, set a new color and didn't see any errors raised in logs